src/components/SupportTA/SupportTA.js

```diff
@@ -0,0 +1,103 @@
+import React, { useEffect, useContext, useMemo } from 'react';
+
+import { Card, CardContent, useContent, useCardState } from 'translation-helps-rcl';
+
+import { AppContext } from '../../context';
+
+export default function SupportTN({
```

Comment:
The name of the function should be SupportTA for coherence. My bad.

Reply:
fixed